plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -398,57 +399,78 @@ void GSSettingsDlg::UpdateControls()
 		EnableWindow(GetDlgItem(m_hWnd, IDC_RESX_EDIT), hw && !integer_scaling);
+		ShowWindow(GetDlgItem(m_hWnd, IDC_RESX_EDIT), hw ? SW_SHOW : SW_HIDE);
+		// Custom resolution Y
 		EnableWindow(GetDlgItem(m_hWnd, IDC_RESY), hw && !integer_scaling);
+		ShowWindow(GetDlgItem(m_hWnd, IDC_RESY), hw ? SW_SHOW : SW_HIDE);
+		// Custom resolution Y Edit
 		EnableWindow(GetDlgItem(m_hWnd, IDC_RESY_EDIT), hw && !integer_scaling);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_CUSTOM_TEXT), hw && !integer_scaling);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_UPSCALE_MULTIPLIER), hw);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_UPSCALE_MULTIPLIER_TEXT), hw);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_PALTEX), hw);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_LOGZ), dx9 && hw);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_FBA), dx9 && hw);
+		ShowWindow(GetDlgItem(m_hWnd, IDC_RESY_EDIT), hw ? SW_SHOW : SW_HIDE);
+		// Upscale multiplier
+		ShowWindow(GetDlgItem(m_hWnd, IDC_UPSCALE_MULTIPLIER), hw ? SW_SHOW : SW_HIDE);
+		ShowWindow(GetDlgItem(m_hWnd, IDC_UPSCALE_MULTIPLIER_TEXT), hw ? SW_SHOW : SW_HIDE);
+		ShowWindow(GetDlgItem(m_hWnd, IDC_PALTEX), hw ? SW_SHOW : SW_HIDE);
 
 		INT_PTR filter;
 		if (ComboBoxGetSelData(IDC_FILTER, filter))
 		{
+			// Anisotropic Filtering
 			EnableWindow(GetDlgItem(m_hWnd, IDC_AFCOMBO), hw && filter && !IsDlgButtonChecked(m_hWnd, IDC_PALTEX));
+			ShowWindow(GetDlgItem(m_hWnd, IDC_AFCOMBO), hw ? SW_SHOW : SW_HIDE);
 			EnableWindow(GetDlgItem(m_hWnd, IDC_AFCOMBO_TEXT), hw && filter && !IsDlgButtonChecked(m_hWnd, IDC_PALTEX));
+			ShowWindow(GetDlgItem(m_hWnd, IDC_AFCOMBO_TEXT), hw ? SW_SHOW : SW_HIDE);
 		}
-		EnableWindow(GetDlgItem(m_hWnd, IDC_ACCURATE_DATE), ogl && hw);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_ACCURATE_BLEND_UNIT), ogl && hw);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_ACCURATE_BLEND_UNIT_TEXT), ogl && hw);
-
-		// Software mode settings
-		EnableWindow(GetDlgItem(m_hWnd, IDC_MIPMAP_SW), sw);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_AA1), sw);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_SWTHREADS_TEXT), sw);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_SWTHREADS_EDIT), sw);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_SWTHREADS), sw);
-
 		// Hacks
-		EnableWindow(GetDlgItem(m_hWnd, IDC_HACKS_ENABLED), hw);
+		ShowWindow(GetDlgItem(m_hWnd, IDC_HACKS_ENABLED), hw ? SW_SHOW : SW_HIDE);
 		EnableWindow(GetDlgItem(m_hWnd, IDC_HACKSBUTTON), hw && IsDlgButtonChecked(m_hWnd, IDC_HACKS_ENABLED));
+		ShowWindow(GetDlgItem(m_hWnd, IDC_HACKSBUTTON), hw ? SW_SHOW : SW_HIDE);
+		//
+
+		// Software mode settings
+		//
+		ShowWindow(GetDlgItem(m_hWnd, IDC_SW_RENDER_TEXT), sw ? SW_SHOW : SW_HIDE);
+		ShowWindow(GetDlgItem(m_hWnd, IDC_MIPMAP_SW), sw ? SW_SHOW : SW_HIDE);
+		ShowWindow(GetDlgItem(m_hWnd, IDC_AA1), sw ? SW_SHOW : SW_HIDE);
+		ShowWindow(GetDlgItem(m_hWnd, IDC_SWTHREADS_TEXT), sw ? SW_SHOW : SW_HIDE);
+		ShowWindow(GetDlgItem(m_hWnd, IDC_SWTHREADS_EDIT), sw ? SW_SHOW : SW_HIDE);
+		ShowWindow(GetDlgItem(m_hWnd, IDC_SWTHREADS), sw ? SW_SHOW : SW_HIDE);
+		//
```

Comment:
I don't really like how `//` has been used as section dividers. If I see an empty comment it makes me think "did they forget to type the comment".